sdks/python/apache_beam/examples/flink/flink_streaming_impulse.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""A streaming workflow that uses a synthetic streaming source.
+
+This can only be used with the Flink portable runner.
```

Comment:
I wonder how meaningful this example is here? It would be nice to convert this into a streaming test case as part of the portable runner tests, e.g. add tests to `flink_runner_test.py`.

Reply:
Not a blocker though, we can do this in a follow-up.

Reply:
It might be useful for others to have this as a ready to run example for profiling?

Reply:
It's definitely useful for us to have the example for benchmarking and cluster validation purposes, but can understand our use cases are somewhat specialized.